### PR TITLE
Update ralph docs

### PR DESCRIPTION
**Pull request by Lbqds**

### CODE REVIEW

File: docs/ralph/getting-started.md [hunk 1]
```diff
@@ -939,3 +1004,22 @@ TxScript Main(fooId: ByteVec) {
   }
 }
 ```
+
+## Gasless Transaction
```
Comment (Lbqds) [inline docs/ralph/getting-started.md:1008]: Not sure if we should put it here.

File: docs/ralph/getting-started.md [hunk 2]
```diff
@@ -939,3 +1004,22 @@ TxScript Main(fooId: ByteVec) {
...
+Contract Foo() {
+  pub fn foo() -> () {
+    payGasFee!(selfAddress!(), txGasFee!())
+  }
+}
+```
+
+The built-in `payGasFee` has two parameters:
+
+1. The first parameter is the payer address, in the example above, the contract paid the gas fee. But the payer address can also be the user addresse.
```
Comment (h0ngcha0) [inline docs/ralph/getting-started.md]:
```suggestion
1. The first parameter is the payer address, in the example above, the contract paid the gas fee. But the payer address can also be the user address.
```

File: docs/ralph/getting-started.md (hunk 1 above)
Comment (h0ngcha0) [reply docs/ralph/getting-started.md:1008]: Alternatively i suppose we can put it under the "Building dApps" section

**Review by h0ngcha0 — APPROVED** [review]
LGTM :+1:

File: docs/ralph/getting-started.md [hunk 3]
```diff
@@ -939,3 +1004,22 @@ TxScript Main(fooId: ByteVec) {
...
 }
 ```
+
+## Gasless Transaction
+
+In the Rhone upgrade, we introduced support for gasless transactions. We can use the built-in `payGasFee` to pay transaction gas fees on behalf of the user, for example:
+
+```rust
+Contract Foo() {
+  pub fn foo() -> () {
```
Comment (polarker) [inline docs/ralph/getting-started.md:1015]: `@using(assetInContract = true)`

File: docs/ralph/getting-started.md [hunk 4]
```diff
@@ -128,6 +128,37 @@ Contract Baz() {
...
+
+    // the variables `x` and `y` already exist, we create two new variables `x1` and `y1`
+    let Foo { x: x1, y: y1 } = foo
+  }
+}
+```
+
 ### Map (Rhone Upgrade Only)
 
 The map data structure is currently available exclusively in the [devnet release](https://github.com/alephium/alephium-stack/tree/master/rhone-devnet) of full node.
```
Comment (polarker) [inline docs/ralph/getting-started.md]: We could update this to both testnet and devnet now

**Review by polarker — APPROVED** [review]
LGTM. minor remarks

File: docs/ralph/getting-started.md (hunk 1 above)
Comment (Lbqds) [reply docs/ralph/getting-started.md:1008]: There are many sub-menu under the `Building dapp` menu, where do you think we should put them?

File: docs/ralph/getting-started.md (hunk 4 above)
Comment (Lbqds) [reply docs/ralph/getting-started.md]: Updated, but there is no `testnet` in `alephium-stack`.

Comment (polarker) [reply docs/ralph/getting-started.md]: Will make one!

**Pull request merged**